src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,37 @@
+.watch {
+  position: relative;
+  box-sizing: border-box;
+  @include size($watch-size, $watch-size);
+  margin: $margin-block auto;
+  border: $watch-border solid $main-dark-color;
+  border-radius: 50%;
+
+  &__center {
+    @include positionCenter;
+    transform: translate(-50%, -50%);
```

Comment:
You use `transform: translate()` everywhere where you use mixin `positionCenter`. You need to remake this mixin in next way - mixin should get parameters for transform property. But these parameters must be optional(default value can be -50%, -50%).